pkg/util/github/github.go

```diff
@@ -193,18 +193,18 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 		tarFile := assetTmpFile[:len(assetTmpFile)-3]
 		err = archive.Ungzip(assetTmpFile, tarFile)
 		if err != nil {
-			return errors.Wrapf(err, "Unable to ungzip %s", assetTmpFile)
+			return errors.Wrapf(err, "Cannot run ungzip %s", assetTmpFile)
 		}
 		err = archive.Untar(tarFile, tmpDir)
 		if err != nil {
-			return errors.Wrapf(err, "Unable to untar %s", tarFile)
+			return errors.Wrapf(err, "Cannot run untar %s", tarFile)
```

Comment:
ugh, I was hoping to avoid a "verbize-noun" (i.e. unzip is not a real verb either). I was thinking if it's a command called "untar" then we would run it. I can't find a lot of usage on this "untar", but if you say it's the accepted verb for it then I guess there is no choice.